packages/blockchain-link-utils/src/solana.ts

```diff
@@ -0,0 +1,43 @@
+import { A, F, pipe } from '@mobily/ts-belt';
+import type { AccountInfo, ParsedAccountData, PublicKey } from '@solana/web3.js';
```

Comment:
Regarding dependencies, how are these handled along the codebase? Although this code works as expected, prettier complains that these `should be listed in the project's dependencies`.

Reply:
follow prettier, each package should list all its dependencies.